docs/navigate/devops-testing/toc.yml

```diff
@@ -38,215 +38,221 @@ items:
-          - name: VB unit testing
-            href: ../../core/testing/unit-testing-visual-basic-with-nunit.md
-          - name: NUnit runner
-            href: ../../core/testing/unit-testing-nunit-runner-intro.md
-      - name: MSTest
-        items:
-          - name: Overview
-            href: ../../core/testing/unit-testing-mstest-intro.md
-          - name: Migrate MSTest from v1 to v3
-            href: ../../core/testing/unit-testing-mstest-migration-from-v1-to-v3.md
-          - name: Getting started
+          - name: xUnit
             items:
-              - name: Overview
-                href: ../../core/testing/unit-testing-mstest-getting-started.md
-              - name: MSTest SDK
-                href: ../../core/testing/unit-testing-mstest-sdk.md
-          - name: Writing tests
+              - name: C# unit testing
+                href: ../../core/testing/unit-testing-with-dotnet-test.md
+              - name: F# unit testing
+                href: ../../core/testing/unit-testing-fsharp-with-dotnet-test.md
+              - name: VB unit testing
+                href: ../../core/testing/unit-testing-visual-basic-with-dotnet-test.md
+              - name: Microsoft.Testing.Platform support in xUnit.net v3
+                href: https://xunit.net/docs/getting-started/v3/microsoft-testing-platform
+              - name: Organize a project and test with xUnit
+                href: ../../core/tutorials/testing-with-cli.md
+                displayName: tutorials, cli
+          - name: NUnit
             items:
-              - name: Overview
-                href: ../../core/testing/unit-testing-mstest-writing-tests.md
-              - name: Examples
-                items:
-                - name: C# example
-                  href: ../../core/testing/unit-testing-with-mstest.md
-                - name: F# example
-                  href: ../../core/testing/unit-testing-fsharp-with-mstest.md
-                - name: VB example
-                  href: ../../core/testing/unit-testing-visual-basic-with-mstest.md
-              - name: Attributes
-                href: ../../core/testing/unit-testing-mstest-writing-tests-attributes.md
-              - name: Assertions
-                href: ../../core/testing/unit-testing-mstest-writing-tests-assertions.md
-              - name: TestContext
-                href: ../../core/testing/unit-testing-mstest-writing-tests-testcontext.md
-          - name: Running tests
+              - name: C# unit testing
+                href: ../../core/testing/unit-testing-with-nunit.md
+              - name: F# unit testing
+                href: ../../core/testing/unit-testing-fsharp-with-nunit.md
+              - name: VB unit testing
+                href: ../../core/testing/unit-testing-visual-basic-with-nunit.md
+              - name: Microsoft.Testing.Platform support in NUnit (aka NUnit runner)
+                href: ../../core/testing/unit-testing-nunit-runner-intro.md
```

Comment:
```suggestion
              - name: Microsoft.Testing.Platform support (NUnit runner)
                href: ../../core/testing/unit-testing-nunit-runner-intro.md
```

Reply:
I think it may be confusing to not mention "in NUnit" in the original statement.
I'm interpreting "Microsoft.Testing.Platform support (NUnit runner)" as if MTP and NUnit runner are the same thing, but they are not. But it's fine to remove "aka" part. What do you think?

Reply:
The reason I removed "in NUnit" it because it's under the NUnit node.

![image](https://github.com/user-attachments/assets/5436d983-cd61-4b50-b075-8447a13767a8)

Reply:
Makes sense. Thanks!